example_notebooks/example_scripts/example_dp.py

```diff
@@ -26,7 +26,7 @@
 
 idr_id = "idr0013"
 tmp_dir = pathlib.Path("../tmp/")
-final_data_dir = pathlib.Path("../example_extracted_features/DP_features")
+final_data_dir = pathlib.Path("../example_extracted_features/DP_features/")
```

Comment:
Just curious, I am seeing that in both example scripts, you are adding a `/` to the ends of the paths. What difference does this make for creating a directory? I haven't been sure what the difference is so I thought I would ask :smile:

Reply:
I believe this is no difference. From now on I am going to try my best to consistently end with a `/` because I think it makes it very clear that this is a path to a directory.